node_modules/datatables/package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "_args": [
+    [
+      "datatables@github:datatables/DataTables#1.10.15",
+      "/home/dbaron/builds/standards-positions"
```

Comment:
Is this something that you are comfortable with?

Reply:
I've pasted enough gdb stack traces into bugzilla that I don't think that aspect of my filesystem structure is a secret, if that's what you're asking.

Should have have invoked `npm` in some other way to avoid that?